src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -1029,6 +1029,39 @@ void Test12()
             VerifyNotInScope(model, x12Ref[1]);
         }
 
+        [Fact]
+        public void PropertyNamedInComplexPattern()
```

Comment:
Sure. Should I hold off on unblocking further IDE testing while I get help with the IDE tests?